### PR TITLE
Remove Choc Autocomplete

### PR DESCRIPTION
**Pull request by NathanielB123**
Group selection actually works consistently now!

### CODE REVIEW

File: frontend/src/Components/EditableFields.tsx [hunk 1]
```diff
@@ -33,31 +31,41 @@ export const EditableField = ({
     });
   };
 
-  const [groupBacking, setGroupBacking] = useState(group);
-
-  const ref = useRef<HTMLInputElement>(null);
-
+  // Given we now are not useing GroupSelect, there is a bit of code duplication
```
Comment (sirjacobofglais) [inline frontend/src/Components/EditableFields.tsx]: nit: using

File: frontend/src/Components/EditableFields.tsx [hunk 2]
```diff
@@ -33,31 +31,41 @@ export const EditableField = ({
...
+          textColor={pickCol(group)}
+          onSubmit={(a) => {
+            handleSubmit(a);
+          }}
+        >
+          <EditablePreview />
+          <EditableInput />
+        </Editable>
       ) : (
+        // <GroupSelect onChange={handleSubmit}>
```
Comment (sirjacobofglais) [inline frontend/src/Components/EditableFields.tsx]: remove this pls

**Review by sirjacobofglais — CHANGES_REQUESTED** [review]

File: frontend/src/Components/EditableFields.tsx (hunk 2 above)
Comment (NathanielB123) [reply frontend/src/Components/EditableFields.tsx]: https://github.com/gpuctl/gpuctl/pull/220/commits/f8760215e76926b4c14b1e26e8fea465acecbf83

File: frontend/src/Components/EditableFields.tsx (hunk 1 above)
Comment (NathanielB123) [reply frontend/src/Components/EditableFields.tsx]: https://github.com/gpuctl/gpuctl/pull/220/commits/f8760215e76926b4c14b1e26e8fea465acecbf83

**Review by sirjacobofglais — APPROVED** [review]

**Pull request merged**